src/pages/transactions/[id].js

```diff
@@ -41,6 +44,13 @@ export default function TransactionDetails() {
         <h2>Description</h2>
         <div>{transaction.description}</div>
       </div>
+      {mode === "delete" && (
+        <Modal
+          message="Really?.. Delete?"
+          onConfirm={() => onDelete(id)}
+          onCancel={onCancel}
+        />
+      )}
```

Comment:
This should be rendered right below the trash bin button instead of at the bottom of the list.